tests/models/test_dagbag.py

```diff
@@ -657,3 +659,35 @@ def test_serialized_dags_are_written_to_db_on_sync(self):
 
             new_serialized_dags_count = session.query(func.count(SerializedDagModel.dag_id)).scalar()
             self.assertEqual(new_serialized_dags_count, 1)
+
+    @patch("airflow.models.dagbag.settings.STORE_SERIALIZED_DAGS", True)
+    @patch("airflow.models.dagbag.settings.MIN_SERIALIZED_DAG_UPDATE_INTERVAL", 5)
+    def test_get_dag_with_dag_serialization(self):
+        """
+        Test that Serialized DAG is updated in DagBag when it is updated in
+        Serialized DAG table after MIN_SERIALIZED_DAG_UPDATE_INTERVAL seconds are passed.
```

Comment:
I think values in the dagbag should update _as soon_ as the serialized dag table changes.

Reply:
Added a new config to control this: https://github.com/apache/airflow/pull/9851/commits/40427e57ab84ca77e2ec60fec4c27001bec7086b